examples/browser/wdio.conf.js

```diff
@@ -2,7 +2,16 @@
 const http = require('http');
 const path = require('path');
 
-const port = 3000;
+const wdioRunnerScript = require.resolve('webdriverio/build/lib/runner.js');
+
+const cliPortKey = '--theia-port';
+const cliPortIndex = process.argv.indexOf(cliPortKey);
+const masterPort = cliPortIndex > -1 ? process.argv[cliPortIndex + 1] : 0;
```

Comment:
Some comments to say how the port is propagated (what you explained to me face to face) would be nice.  Otherwise it's a real maze.